transmission_interface/src/differential_transmission_loader.cpp

```diff
@@ -239,6 +243,17 @@ bool DifferentialTransmissionLoader::getJointConfig(const TransmissionInfo& tran
     if (offset_status == BAD_TYPE) {return false;}
   }
 
+  // Parse if transmision has to be ignored for absolute encoders
+  ignore_transmission_for_absolute_encoders = true;
+  for (unsigned int i = 0; i < 2; ++i){
+    const TiXmlElement* role_el = jnt_elements[i].FirstChildElement("ignoreTransmissionAbsoluteEncoder");
+    bool ignore = false;
+    if(role_el){
+      ignore = true;
+    }
+    ignore_transmission_for_absolute_encoders &= ignore;
```

Comment:
This is redundant. Can just be:

`ignore_transmission_for_absolute_encoders &=static_cast<bool>(role_el);` or `(role_el != nullptr)` etc.

Reply:
it feels like I've fixed this a 100 times already, thanks